packages/documentation/src/stories/patterns/forms/forms.stories.ts

```diff
@@ -0,0 +1,349 @@
+    <div class="row row-cols-1 row-cols-md-2">
+      <div class="col">
+        <label class="form-label" for="firstname">Firstname</label>
+        <input type="text" id="firstname" class="form-control" />
+      </div>
+      <div class="col">
+        <label class="form-label" for="lastname">Lastname</label>
+        <input type="text" id="lastname" class="form-control" />
+      </div>
+    </div>
+  `,
+};
+
+export const ColSimple: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-8">
+        <label class="form-label" for="city">City</label>
+        <input type="text" id="city" class="form-control" />
+      </div>
+      <div class="col-4">
+        <label class="form-label" for="state">State</label>
+        <input type="text" id="state" class="form-control" />
+      </div>
+    </div>
+  `,
+};
+
+export const CustomWidth: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-auto">
+        <label class="form-label" for="zip">Zip</label>
+        <input type="number" id="zip" class="form-control" style="max-width: 11ch" />
+      </div>
+      <div class="col">
+        <label class="form-label" for="city">City</label>
+        <input type="text" id="city" class="form-control" />
+      </div>
+    </div>
+  `,
+};
+
+export const VerticalSpacing: Story = {
+  render: () => html`
+    <div class="mb-regular">
+      <label for="firstname">Firstname</label>
+      <input type="text" id="firstname" class="form-control" />
+    </div>
+    <div class="mb-regular">
+      <label for="lastname">Lastname</label>
+      <input type="text" id="lastname" class="form-control" />
+    </div>
+  `,
+};
+
+export const Buttons: Story = {
+  render: () => html`
+    <div class="mb-regular">
+      <label for="firstname">Firstname</label>
+      <input type="text" id="firstname" class="form-control" />
+    </div>
+    <div class="d-flex flex-row-reverse gap-mini">
+      <button class="btn btn-primary">Send</button>
+      <button class="btn btn-secondary">Cancel</button>
+    </div>
+  `,
+};
+
+export const FloatingLabels: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col">
+        <div class="form-floating">
+          <input type="text" id="firstname" class="form-control" placeholder=" " />
+          <label class="form-label" for="firstname">Firstname</label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-floating">
+          <input type="text" id="lastname" class="form-control" placeholder=" " />
+          <label class="form-label" for="lastname">Lastname</label>
+        </div>
+      </div>
+    </div>
+  `,
+};
+
+export const Validation: Story = {
+  render: () => html`
+    <form action="/" class="was-validated">
+      <!-- Form Control -->
+      <div class="row mb-regular">
+        <div class="col">
+          <div class="form-floating">
+            <input
+              id="FormControlInvalid"
```

Comment:
Just a detail but it would be better to use camel case for ids:
```suggestion
              id="formControlInvalid"
```